m/configDTOutputNetCDFL2.m

```diff
@@ -620,7 +648,131 @@
     'grid_coordinates' []
     'grid_method'      [] };
 
-
+  %% Variables specific to SeaExplorer
+  var_attr_list.mfl_v1 = {
+      'long_name'     'MiniFluo raw path 1'
+      'standard_name' 'tryptophan_or_pyrene_measurement'
+      'units'         'counts'
+      'coordinates'   'time depth latitude longitude'
+      '_FillValue'    default_fill_value
+      'sources'       []
+      'conversion'       []
+      'grid_sources'     []
+      'grid_coordinates' []
+      'grid_method'      [] };
+  
+  var_attr_list.mfl_v2 = {
+      'long_name'     'MiniFluo raw path 2'
+      'standard_name' 'phenantrene_or_fluoren_measurement'
```

Comment:
Ok, let's put `fluorene` for now, but I am just coming back from a meeting and these variable names should change. Variable names should include emission/detection wavelength used instead of target component since other fluorophores may also be excited.  It will be up to the user to decide what it represent. I will update this in a future version.